fe/fe-core/src/main/java/com/starrocks/lake/LakeTablet.java

```diff
@@ -55,13 +56,19 @@ public class LakeTablet extends Tablet {
     private static final String JSON_KEY_DATA_SIZE = "dataSize";
     private static final String JSON_KEY_ROW_COUNT = "rowCount";
     private static final String JSON_KEY_DATA_SIZE_UPDATE_TIME = "dataSizeUpdateTime";
+    private static final String JSON_KEY_PK_INDEX_MEM_SIZE = "primaryIndexMem";
+    private static final String JSON_KEY_PK_INDEX_DISK_SIZE = "primaryIndexDisk";
 
     @SerializedName(value = JSON_KEY_DATA_SIZE)
```

Comment:
I'm curious that, in Database class, the serialized names are single character, but in Tablet class (like LocalTablet), the serialzed names are full words, will it affect the meta image size a lot?